mozilla-mobile/firefox-ios
# Add FXIOS-8098, DISCO-2666 [v123] Hide sponsored suggestions from

bookmarks and history




## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-8098)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/18018)

## :bulb: Description
We want to filter out history, bookmarks and synced tab suggestions for URLS that contain the sponsored advertiser query param. I believe that desktop does this with a pref and I tried to do the same here.

Steps to verify:
- Ensure Firefox Suggest is enabled and "Suggestions from Sponsors" is enabled
- query "ama" in the url bar and click on sponsored suggestion
- query "ama" in the url bar again verify that under Firefox Suggest none of the suggestions have the "mfadid=adm" param

## :pencil: Checklist
You have to check all boxes before merging
